Microsoft.Identity.Web/TokenCacheProviders/Sql/SqlTokenCacheProviderExtension.cs

```diff
@@ -22,37 +20,6 @@ public static class SqlTokenCacheProviderExtension
         public static IServiceCollection AddSqlTokenCaches(
```

Comment:
nit: Can this be singular? All the others are singular (i think)...ex `AddSqlTokenCache`